dotnet/src/dotnetframework/GxClasses/Services/GxRestWrapper.cs

```diff
@@ -537,6 +537,16 @@ public bool IsAuthenticated(string synchronizer)
 					SetError("0", "Invalid Synchronizer " + synchronizer);
 			}
 		}
+		public bool IsAuthenticatedMethod(string serviceMethod, bool isApi)
```

Comment:
If possible, make this method private.

Reply:
Changed to protected. Private does not work when called from generated code.